core/src/main/scala/cats/std/bigInt.scala

```diff
@@ -1,7 +1,14 @@
 package cats
 package std
 
-trait BigIntInstances extends algebra.std.BigIntInstances {
+trait BigIntInstances {
+  implicit val bigIntAlgebra: BigIntAlgebra =
```

Comment:
I was just proposing `: Order[BigInt]` instead of `: BigIntAlgebra` for the return type. As far as I know that's the approach we are taking pretty much everywhere else.